src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -322,7 +322,7 @@ protected RequestExecutor(string databaseName, X509Certificate2 certificate, Doc
 
             ClientConfigurationChanged += (_, args) =>
             {
-                if (args.Configuration.ReadBalanceBehavior == ReadBalanceBehavior.FastestNode)
+                if (args.Configuration?.ReadBalanceBehavior == ReadBalanceBehavior.FastestNode)
```

Comment:
shouldn't this be:

```
if (args.Configuration == null) // backward compatibility
    return;

// old code
```

Reply:
my though was that if the config is `null` as if no configuration, I do want to disable the fastest node read balance